clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -135,6 +135,11 @@
                     "description": "Default list of USS paths to search for copybooks\nZowe Explorer version 1.15.0 or higher is required to download copybooks from the mainframe",
                     "uniqueItems": true
                 },
+                "cobol-lsp.cpy-manager.paths-uss-encoding": {
```

Comment:
@zeibura 

![image](https://user-images.githubusercontent.com/69206564/151131266-4c45942e-4167-4569-8401-b90478c0e1d9.png)

Reply:
updated to "cobol-lsp.cpy-manager.uss-copybook-in-ibm-1047"